python/doc/pyplots/P1LagrangeEvaluation.py

```diff
@@ -2,18 +2,30 @@
 from matplotlib import pyplot as plt
 from openturns.viewer import View
 
```

Comment:
none of these graphs show up probably because of missing `:template: classWithPlot.rst_t`

Reply:
I added it!